src/onegov/election_day/templates/archive.pt

```diff
@@ -192,8 +192,10 @@
                             <a tal:condition="python: str(year) != date" href="${request.link(layout.archive.for_date(year))}">${year}</a>
                         </li>
                     </ul><br>
-                    <p ><a i18n:translate href="${layout.archive_search_link}">Archive Search</a></p>
-                    <p ><a i18n:translate href="${layout.archive_download}">Download the entire archive</a></p>
+                    <p><a i18n:translate href="${layout.archive_search_link}">Archive Search</a></p>
+                    <div tal:define="timestamp layout.last_archive_modification" >
+                        <p tal:condition="timestamp"><a i18n:translate href="${layout.archive_download}">Download the entire archive (Status: <tal:block tal:content="layout.format_date(timestamp, 'datetime')" i18n:name='date'/>)</a></p>
```

Comment:
This way the option of downloading the archive does not appear when the archive has never been downloaded before. I wold put the Status in a `<tal:b>` and set the condition there.